docs/install.md

```diff
@@ -131,27 +131,44 @@ page. Copy the contents of the SPID into the file
 
 SGX can run in either simulation or hardware mode. No kernel driver is
 required to run in simulation mode. However, if you plan to run with SGX
-hardware support, it is necessary to install the SGX kernel driver. The
+hardware support, it is necessary to install the SGX kernel driver. 
+
+<!-- DCAP kernel driver installation -->
+The following commands will download and install the driver version 1.3 of
+the DCAP SGX kernel driver (for Ubuntu 18.04 server):
+
+```bash
+DRIVER_REPO=https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu18.04-server/
+DRIVER_FILE=sgx_linux_x64_driver_1.35.bin
+
+wget ${DRIVER_REPO}/${DRIVER_FILE} -P /tmp
+chmod a+x /tmp/${DRIVER_FILE}
+sudo ./${DRIVER_FILE}
+```
+Note:
+- as of early August 2020, the sdk drivers (e.g., `sgx_linux_x64_driver_2.6.0_602374c.bin`) will cause BUS errors in some circumstance and is currently not supported by PDO
+- the DCAP driver, though, requires hardware which supports FLC (Flexible Launch Control)
```

Comment:
Minor thing: Should we add a comment here about how we have actually tested the DCAP driver on HW with FLC?

Reply:
did you have something specific in mind? Not sure exactly what to add. Do you mean something like reference [Validate the installation](https://github.com/hyperledger-labs/private-data-objects/blob/master/docs/usage.md#validate-the-installation) and say do it in HW mode? Or something DCAP specfic?

Reply:
Ah, I didn't realize we had this in our docu! In that case, yes I think it would good to reference this validation step and specify to do it in HW mode (and maybe also show how to do the equivalent in docker).

Reply:
Ok, added now a section on validation for HW mode